config/application.rb

```diff
@@ -33,5 +33,11 @@ class Application < Rails::Application
 
       g.fallbacks[:rspec] = :test_unit
     end
+
+    # Configure webpack
+    config.webpack.config_file = "config/webpack.js"
+    config.webpack.output_dir  = "public/assets/webpack"
+    config.webpack.public_path = "assets/webpack"
+    config.webpack.dev_server.enabled = false
```

Comment:
I saw that webpack-rails README recommends to use foreman to start both rails server and webpack-dev-server. Also checked [helper.rb](https://github.com/mipearson/webpack-rails/blob/master/lib/webpack/rails/helper.rb#L10) that says if not using dev server, it will rely on precompiled assets. But that's production behaviour, right? What about development?